client/src/main/java/io/pravega/client/stream/impl/ReaderGroupImpl.java

```diff
@@ -92,6 +93,10 @@
     private final SegmentMetadataClientFactory metaFactory;
     private final StateSynchronizer<ReaderGroupState> synchronizer;
     private final NotifierFactory notifierFactory;
+    @VisibleForTesting
+    private final ConnectionPool connectionPool;
+    @VisibleForTesting
+    private final SequentialProcessor ackProcessor;
```

Comment:
Please call this something else as this name implies it does something quite different.

Reply:
Sure, Taken care